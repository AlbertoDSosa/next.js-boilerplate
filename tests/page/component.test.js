import React from 'react';

import Prueba from './component';
import {shallow} from 'enzyme';

describe('Testing enzyme snapshot components', () => {
  test('Shallow component' , () => {
    const wrapper = shallow(<Prueba />);

    expect(wrapper).toMatchSnapshot();
  })
});