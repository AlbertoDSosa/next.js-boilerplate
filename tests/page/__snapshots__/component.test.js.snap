// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing enzyme snapshot components Shallow component 1`] = `
<div>
  Hola
</div>
`;
